src/uct/ib/base/ib_device.c

```diff
@@ -18,6 +18,21 @@
 #include <sched.h>
 
 
+/* use both gid + lid data for key generarion (lid - ib based, gid - RoCE) */
+#define kh_ah_hash_func(_key) kh_int64_hash_func((_key).attr.grh.dgid.global.subnet_prefix ^ \
```

Comment:
uct_ib prefix: uct_ib_kh_ah_hash_func

also, can we use inline function instead of macro?